cmd/mimir/config-descriptor.json

```diff
@@ -8328,7 +8328,6 @@
             "Host": "localhost:8080",
             "Path": "/alertmanager",
             "RawPath": "",
-            "OmitHost": false,
```

Comment:
This change will break in CI. I think the change happens because you have an older Go version.

Reply:
updated my go sdk